doc/install_linux_prerequisites.md

```diff
@@ -20,13 +20,3 @@ Install the latest Python 3.9 available in the software repo.
 ```
 sudo dnf install -y gcc libffi-devel python39-devel openssl-devel
 ```
-
-### SUSE OpenSUSE 13.2
-Install Python 3.8+ if needed.
-```
-sudo zypper refresh && sudo zypper --non-interactive install gcc libffi-devel python-devel openssl-devel
-```
-
-### Flatcar
-
-Python is installed in the Azure-specific distribution of Flatcar, but is installed into the non-standard location `/usr/share/oem/python/bin/python`.
```

Comment:
We've dropped SUSE， ref: https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?branch=live&pivots=zypper#before-you-begin

We don't create package for flatcar, I don't see a reason to mention it here.

PS: This doc is about install CLI with `pip`, which is not recommended. I'm considering removing it.